### ISSUE TITLE
Enforge selection from suggested words only?

### ISSUE THREAD

**Issue by itajackass** [pawelczak/EasyAutocomplete#326]
Is there a way to enforce selection of a item only if is in the list?
For example if the list is:

CANA
CANADA
CANAPA
CANNES

and i wrote only CA and I focus out, input field must be set to empty....but if i select an item suggested, is ok.

**Comment by sic-sic** [comment]
This would be a very nice feature. I currently need it.
Any solution until now?

**Comment by adarshmadrecha** [comment]
Yes, this feature would really help, or at least there could be an event which we can listen to in case the user did not select from the list.

**Comment by xileftenurb** [comment]
I think you could use the "onChooseEvent", check the current selected Item, and set to empty if it is not in the list.

**Comment by Hissvard** [comment]
Yeah, I did exactly what @xileftenurb suggested. Just listening for the change event and emptying the input in case it's not in the list of options.
It does the trick as a workaround, but it's definitely something I can see being needed often so maybe it should be builtin functionality.

**Comment by lguariento** [comment]
This is really a feature which would be greatly appreciated as a built-in :) . Bump!

**Comment by kallewesterling** [comment]
I will also bump this :)

**Comment by kristenzirkler** [comment]
Same!

**Comment by jarodium** [comment]
Same!
